roles/yarn/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: install node.js
+  include_role:
+    name: geerlingguy.nodejs
+  vars:
+    nodejs_version: "{{ node_version }}"
+    npm_config_prefix: "{{ npm_prefix }}"
```

Comment:
[nodenv](https://github.com/nodenv/nodenv) is the Node's equivalent of `rbenv` and we'll need it when upgrading node versions, as we do with Ruby. I don't mind starting with this, but at some point we'll need to manage multiple versions.